cmd/junitreport/pkg/api/test_case.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"strings"
+	"time"
+)
+
```

Comment:
@soltysh @liggitt Are we saying that this needs to change so no serialized objects have methods? We will just have a package of mutating methods for the current contents of `string.go`, *etc*?